tsconfig.json

```diff
@@ -20,5 +20,8 @@
   "include": [
     "./src/**/*.js",
     "./types"
+  ],
+  "exclude": [
+    "./types/tests"
```

Comment:
`types/tests` removed from the config used by `lint-tsc`, because not all `// @ts-except-error` directives trigger without `"strict: true"` and the `src` folder gives a lot of errors with strict enabled.

types are tested with `lint-types`, using the config in `types/tests`:
`"lint-types": "eslint \"types/**/*.ts\" && tsc -p types/tests/",`